be/src/vec/functions/array/function_array_apply.cpp

```diff
@@ -64,6 +64,8 @@ class FunctionArrayApply : public IFunction {
     size_t get_number_of_arguments() const override { return 3; }
     ColumnNumbers get_arguments_that_are_always_constant() const override { return {1, 2}; }
 
+    bool dont_append_return_type_name_when_register_function() const override { return true; }
```

Comment:
use `has_return_type_in_signature` as name and use the opposite in values and usages